OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -131,6 +131,10 @@ public override bool Tick(Actor self)
 					target = Target.FromCell(self.World, newLocation.Value);
 					targetPosition = target.CenterPosition + offset;
 					landingCell = self.World.Map.CellContaining(targetPosition);
+
+					// Since the landing cell have changed, check again; if we are already at the new landing location, then abort landing
```

Comment:
Nit: The period went missing.
```suggestion
					// Since the landing cell have changed, check again; if we are already at the new landing location, then abort landing.
```

Reply:
Is the comment even needed? The code already reads like this

Reply:
I see your point. I am fine with either the typo fix or removal of the comment altogether.

Reply:
I removed the comment.